locales/hin.json

```diff
@@ -384,7 +387,7 @@
         },
         "accepted": {
           "title": "सफलता!",
-          "message": "आपका {{vcLabel}} को {{रिसीवर}} के साथ सफलतापूर्वक साझा कर दिया गया है"
```

Comment:
Translation for {{vclabel}} and {{receiver}} is missing

Reply:
This translation SendVcScreen:status is not available in en.json locale file. Only the translations that were available in en.json were translated.

Reply:
Check with bhargavi, all the missed translation have been added. if it is used and missed translations,
1. please add the needed translations .
2. if it not used, please remove the unused literals.

Reply:
if it is used and missed translations -> please add the needed translations => This has been done in this PR.
For removing unused literals, will connect with @PuBHARGAVI once and will do the needful

Reply:
Update: all the {{vcLabel}} has been translated in all locales file except English